modules/sr/robot/ruggeduino.py

```diff
@@ -34,7 +34,18 @@ def init_ruggeduino_array(webot: Robot) -> 'List[Ruggeduino]':
     digital_input_array = [Microswitch(webot, name) for name in switch_names]
 
     limiter = OutputFrequencyLimiter(webot)
-    digital_output_array = [Led(webot, name, limiter) for name in led_names]
+    digital_output_array = [
+        Led(
+            webot,
+            name,
+            limiter,
+            pin,
+        )
```

Comment:
Miiight be worth unwrapping this now?
```suggestion
        Led(webot, name, limiter, pin)
```